### PR TITLE
Set default value of `NRLoggingEnabled` Parameter to True

### PR DESCRIPTION
**Pull request by AkhilReddyS-NR**
- Changed the default value of the Parameter `NRLoggingEnabled` to True
- This change eases the deployment process for customers by removing the need to manually configure logging .
- Details of testing done
       Manual Testing:
                - Manually tested the Lambda function with the `loggingEnabled` variable set to true.
                - Verified that logs are being forwarded to NewRelic.
                - Confirmed the function's behavior and performance with logging enabled and logging disabled.
                - Ensured no unintended behavior or issues were introduced.
       Automated Testing:
               - Executed all the test cases present in the `tests` folder of the repository.
               - Ensured that all tests passed successfully without any errors or failures.
- Jira story - [NR-303954](https://new-relic.atlassian.net/browse/NR-303954)

[NR-303954]: https://new-relic.atlassian.net/browse/NR-303954?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by maya-jha** [comment]
1. Can you add details of testing done.
2. Also add the Jira/github issue id.

File: serverless.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ functions:
   newrelic-log-ingestion:
     description: Send log data from CloudWatch Logs to New Relic Infrastructure (Cloud Integrations) and New Relic Logging.
     handler: src/function.lambda_handler
-    name: newrelic-log-ingestion
+    name: newrelic-log-ingestion-akhilv1
```
Comment (maya-jha) [inline serverless.yml:39]: why this name change?

Comment (AkhilReddyS-NR) [reply serverless.yml:39]: Thanks for catching that. This was an unintended change made during my testing. I will revert the function name to its original value and push the correction shortly

**Comment by AkhilReddyS-NR** [comment]
> 1. Can you add details of testing done.
> 2. Also add the Jira/github issue id.

Added the details of testing and Jira Id to the desription.

**Comment by maya-jha** [comment]
Can you update README.md and remove
```
* To set `LOGGING_ENABLED`: `sam build ... --parameter-overrides 'ParameterKey=NRLoggingEnabled,ParameterValue=True'`
* To set `LOGGING_ENABLED`: `export LOGGING_ENABLED=True` (prior to deploy)```

**Review by maya-jha — APPROVED** [review]
lgtm.

**Pull request merged**